charts/celeborn/templates/_helpers.tpl

```diff
@@ -155,3 +155,78 @@ Create the name of the worker podmonitor to use
 {{- define "celeborn.workerPodMonitorName" -}}
 {{ include "celeborn.fullname" . }}-worker-podmonitor
 {{- end }}
+
+{{/*
+Create master annotations if metrics enables
+*/}}
+{{- define "celeborn.masterMetricsAnnotation" -}}
+{{- $metricsEnabled := true -}}
+{{- $metricsPath := "/metrics/prometheus" -}}
+{{- $masterPort := 9098 -}}
+{{- range $key, $val := .Values.celeborn }}
+{{- if eq $key "celeborn.metrics.enabled" }}
+{{- $metricsEnabled = $val -}}
+{{- end }}
```

Comment:
I have a question about helm chart indentation. Is there any guideline or best practice?

Reply:
Thanks for you review. The file `_helpers.tpl` is go template syntax, some complex logic can be coded in this file, I understand. The param `celeborn.metrics.enabled`, and others,  is written in values.yaml as a single key, but the dot `.` inside key does not conform to the specification which cannot be used directly by `.Values.xxx`. So, I imitate the way in `configMap.yaml`, using the `range` syntax to loop through, and using if to scratch the value. The indentation and format in this file is not significant, and it is processed through variable  in template/xx/service.yaml.
![image](https://github.com/user-attachments/assets/c3148fa3-ecc8-4a9f-a996-408daf9eef32)

Reply:
> I have a question about helm chart indentation. Is there any guideline or best practice?

sorry for not link your review.